process/process.go

```diff
@@ -209,6 +218,33 @@ func loadProcess(ctx context.Context, pid int) (*Process, error) {
 	}
 	process.ParentPid = int(ppid)
 
+	// Get all environment variables
+	envVariables, err := pInfo.EnvironWithContext(ctx)
+
+	// Not important enough to stop execution of the function
+	if err == nil {
+		// Try to find PORTMASTER_PROFILE variable
+		for _, envVar := range envVariables {
+			if strings.HasPrefix(envVar, profileEnvironmentVariable) {
+				// Found. Check validity and set to process
+				splitted := strings.SplitN(envVar, "=", 2)
+				if len(splitted) == 2 {
+					if hasNoSpecialCharacters(splitted[1]) {
+						process.EnvironmentProfileID = splitted[1]
+						log.Infof(`using "%s" profile id for process: %d`, process.EnvironmentProfileID, process.Pid)
+					} else {
+						log.Warningf(`invalid %s value: %q`, profileEnvironmentVariable, splitted[1])
+					}
+				} else {
+					log.Warningf(`empty value of %s`, profileEnvironmentVariable)
+				}
+				break
+			}
+		}
+	} else {
+		log.Errorf("failed to read environment variables: %q", err)
```

Comment:
This and others: Please log to context and prefix the message with the module name. Eg.
`log.Tracer(ctx).Errorf("process: failed to read environment variables: %s", err)`